src/backend/graphItemService.ts

```diff
@@ -3,11 +3,16 @@ import {GroupedErrors, parseCSV, RowErrorMessage} from './utils';
 import {ImportEdgesParams, ImportItemsResponse, ImportNodesParams} from '../@types/shared';
 
 export class GraphItemService {
-  public static async importGraphItems(
+
+  public importResult!: ImportItemsResponse;
```

Comment:
importResult can be undefined, imagine if you call the API before the import is started.